br/pkg/lightning/config/config.go

```diff
@@ -100,6 +100,7 @@ const (
 
 	DefaultEngineMemCacheSize      = 512 * units.MiB
 	DefaultLocalWriterMemCacheSize = 128 * units.MiB
+	DefaultBlockSize               = 16 * units.KiB
```

Comment:
can you update this setting for `import into`
https://github.com/pingcap/tidb/blob/cf63fddd9b8491af6374e9eac36ca48f5640ba36/pkg/executor/importer/import.go#L1318-L1319

Reply:
modified